pocs/core.py

```diff
@@ -372,8 +372,10 @@ def is_weather_safe(self, stale=180):
             timestamp = record['date'].replace(tzinfo=None)  # current_time is timezone naive
             age = (current_time().datetime - timestamp).total_seconds()
 
-            self.logger.debug(
-                "Weather Safety: {} [{:.0f} sec old - {}]".format(is_safe, age, timestamp))
+            self.logger.debug("Weather Safety: {} [{:.0f} sec old - {:%d-%m-%Y %H:%M:%S}]",
```

Comment:
Wot no ISO 8601? ;-)

Shouldn't it be `{:%Y-%m-%d %H:%M:%S}` to format the date-time as in the PR description?

Reply:
Uh, yes, it should be. I was focus on the microseconds. :)